.github/workflows/ci-release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release CI
+
+"on":
+  push:
+    branches-ignore:
+      # These should always correspond to pull requests, so ignore them for
+      # the push trigger and let them be triggered by the pull_request
+      # trigger, avoiding running the workflow twice.  This is a minor
+      # optimization so there's no need to ensure this is comprehensive.
+      - "dependabot/**"
+      - "renovate/**"
+      - "tickets/**"
+      - "u/**"
+    tags:
+      - "*"
+  pull_request: {}
```

Comment:
This was to get an initial workflow in place so we could do testing.  To show and run in GitHub the workflow needs to be pushed to main.  There will be changes.  This can replace `build-service`, but didn't want to do that before testing was completed.

Reply:
Sorry, reading this again I don't follow. You seem to have already tested this before putting it in review (https://github.com/lsst-dm/prompt_processing/pkgs/container/prompt_processing shows the PR build, after all), and without pushing to `main`. Isn't the only part that must be tested on `main` the actual tag use?

Given your later remark about "co-existence", and the absence of a Phalanx PR, I feel like I'm missing something fundamental about your goal. Would it be useful to have a quick meeting on Zoom or Slack huddle? I have a meeting from 2-3 PST (4-5 CST) but am otherwise free.

Reply:
Thanks.  Later in the day for me now.  Would a time work tomorrow?  I am free before noon PST or we could stay on the standup?

Reply:
After standup works for me.

Reply:
We discussed this this afternoon -- the plan was to push the new workflow and Dockerfile alongside the old ones, then leave them there for an evaluation period while the old files remained untouched. They would then be consolidated on a later PR also associated with DM-40418. This goes against DM workflow, but it does explain the main issues I was confused about, so I won't hold up merge over it.